### ISSUE TITLE
Users need to receive the renewal notification long before the permit expires

### ISSUE THREAD

**Issue by AbiBarkerOxfordshire** [alphagov/verify-local-patterns#450]
Impact: Users might forget to renew their permit before is expires and then get a ticket.
Evidence: “Would like a reminder well in advance. Need enough time to renew.”
“Don’t remind in 12 months, that’s too late!”

**Comment by EUzkuraityte** [comment]
Same issue as #331

**Comment by sanjaypoyzer** [comment]
@EUzkuraityte Not sure how this different to #331?

**Comment by suryanaga** [comment]
Need to explore how much time is enough time to renew here. 30 days seems reasonable to me.

**Comment by EUzkuraityte** [comment]
@sanjaypoyzer Yup, sound reasonable to me. Can you can remove the code from the page below (that pulls user's choice re: permit length) and replace it with advice that the user knows they will receive renewals notification 30 days before the permit expires. 
@lizziebruce please can have a look and let us know the best way to phrase this advice. 

![screen shot 2017-05-26 at 10 20 23](https://cloud.githubusercontent.com/assets/20285115/26488891/12b73e70-41fd-11e7-94e3-7aa387ce31f4.png)

**Comment by lizziebruce** [comment]
@EUzkuraityte @sanjaypoyzer 

Should we also add in a postal option?  

> **Renewal reminder**
> 
> We can send you a renewal reminder 30 days before your parking permit expires.
>
> **How do you want to receive your renewal reminder?**
> 
> [ ] Email
> 
> [ ] Text message
>
> [ ] By post

**Comment by EUzkuraityte** [comment]
@lizziebruce yes, definitely - this looks great to me
